07_deploy_masters.sh

```diff
@@ -29,6 +29,9 @@ for node in $(jq -r .nodes[].name ${MASTER_NODES_FILE}); do
   fi
 done
 
+echo "Have fun playing with test-terraform.sh !"
+exit 0
```

Comment:
Could we just directly run the terraform script from here, maybe gated by an env variable?

Reply:
Also the script is "terraform-test.sh"

Reply:
I'd say lets just run terraform here, as a step towards running it via kni-install?

Reply:
Totally, shouldn't take much more work to just have 07_deploy_masters.sh use terraform - I just pushed this as WIP for playing with

Reply:
Ok, switched over to terraform here - it's kicking off the node deployments ok, but I suspect the igntion config isn't getting through - haven't debugged yet

Reply:
Ah, yea, it looks like it's not getting through, I think it might be a bug in the provider, looking into it

Reply:
So, it was my misunderstanding of how `DiffSurpressFunc` worked in Terraform.  It should work now, or at least it actually builds the config drive.